source/_integrations/utility_meter.markdown

```diff
@@ -154,6 +154,20 @@ Calibrate the Utility Meter. Change the value of a given sensor.
 | `entity_id` | no | String or list of strings that point at `entity_id`s of utility_meters.
 | `value` | no | Number | Value to calibrate the sensor with | 
 
+### Service `select.select_option`
```

Comment:
This service is not part of this integration, and thus should not be documented on this page.

Nevertheless, thanks for willing to contribute 👍

../Frenck